uber/utils.py

```diff
@@ -477,6 +477,31 @@ def get_real_badge_type(badge_type):
     return c.ATTENDEE_BADGE if badge_type in [c.PSEUDO_DEALER_BADGE, c.PSEUDO_GROUP_BADGE] else badge_type
 
 
+def add_opt(opts, other):
+    """
+    Add an option to a comma-separated string.
+    """
+    other = [other] if isinstance(other, str) else other
```

Comment:
In general, this is not a great way to check if an object is a string, because it fails against byte strings. You could do ` isinstance(other, (str, bytes))` for python 3, or ` isinstance(other, (str, unicode))` for python 2, or be portable and do ` isinstance(other, six.string_types)`.

Fortunately `sideboard` has you covered! This particular construction is so common, there's a helper method for it!
```python
other = listify(other)
```

Also, if `None` is passed in for `other`, you'll end up with a result like this `"21907321,None"`. And if an empty string is passed in, you'll end up with a trailing comma, like this `"21907321,"` (which might be fine?). So you probably want to do something like this:
```python
other = listify(other) if other else []
```

I know you've added the `set()` uniquification code in the `MultiChoice` class itself, but I think it would be safer to also add it here. I don't think `process_bind_paran()` is called until the object is saved to the database, so you could potentially end up rendering the same ribbon twice in a template somewhere (or something).

How about something like this?
```python
def add_opt(opts, other):
    other = set(listify(other) if other else [])
    list = opts.split(',') if opts else []
    list.extend(other)
    return ','.join(list)
```